packages/gestalt/src/ComboBoxItem.js

```diff
@@ -0,0 +1,93 @@
+// @flow strict
+import { forwardRef, type Node } from 'react';
+import Text from './Text.js';
+import Icon from './Icon.js';
+import focusStyles from './Focus.css';
+
+export type OptionItemType = {|
```

Comment:
Should this type name be updated to match the component/file name? (I don't feel strongly about this, just want to make sure it's intentional)

Reply:
DONE